### DISCUSSION TITLE
no output from level-2

### DISCUSSION THREAD

**Discussion by alwadey** [davidfrantz/force#308] [Q&A] [answered]
I am trying to produce force level 2 tiles from 2 sentinel-2 scenes using this commend :

force-level2 /home/alwadey/force/param/short.prm 

using the flowing .prm file no output is produced but the logs show that  :

/home/alwadey/force/level1/T38RPN/S2B_MSIL1C_20220125T073109_N0400_R049_T38RPN_20220125T093037.SAFE
: dc: 100.00%. wc:   0.00%. sc:   0.00%. cc: 100.00%. Skip. Processing time: 01 mins 29 secs

in the terminal, it is shown :
1 images enqueued. Start processing with 1 CPUs
No valid parameter file!
Parsing parameter file failed.

Computers / CPU cores / Max jobs to run
1:local / 8 / 1

Computer:jobs running/jobs completed/%of started jobs/Average seconds to complete
ETA: 0s Left: 1 AVG: 0.00s  local:1/0/100%/0.0s No valid parameter file!
Parsing parameter file failed.
ETA: 0s Left: 0 AVG: 0.00s  local:0/1/100%/88.0s 

The .prm:

++PARAM_LEVEL2_START++

# INPUT/OUTPUT DIRECTORIES
# ------------------------------------------------------------------------
FILE_QUEUE = /home/alwadey/force/level1/queue.txt
DIR_LEVEL2 = /home/alwadey/force/level2
DIR_LOG = /home/alwadey/force/log
DIR_TEMP = /home/alwadey/force/temp

# DIGITAL ELEVATION MODEL
# ------------------------------------------------------------------------
FILE_DEM = NULL
DEM_NODATA = -32767

# DATA CUBES
# ------------------------------------------------------------------------
DO_REPROJ = FALSE
DO_TILE = FALSE
FILE_TILE = NULL
TILE_SIZE = 30000
BLOCK_SIZE = 3000
RESOLUTION_LANDSAT = 30
RESOLUTION_SENTINEL2 = 10
ORIGIN_LON = -25
ORIGIN_LAT = 60
PROJECTION = GLANCE7
RESAMPLING = NN

# RADIOMETRIC CORRECTION OPTIONS
# ------------------------------------------------------------------------
DO_ATMO = TRUE
DO_TOPO = FALSE
DO_BRDF = TRUE
ADJACENCY_EFFECT = TRUE
MULTI_SCATTERING = TRUE

# WATER VAPOR CORRECTION OPTIONS
# ------------------------------------------------------------------------
DIR_WVPLUT = NULL
WATER_VAPOR = NULL

# AEROSOL OPTICAL DEPTH OPTIONS
# ------------------------------------------------------------------------
DO_AOD  = TRUE
DIR_AOD  = NULL

# CLOUD DETECTION OPTIONS
# ------------------------------------------------------------------------
MAX_CLOUD_COVER_FRAME = 75
MAX_CLOUD_COVER_TILE  = 75
CLOUD_THRESHOLD  = 0.225
SHADOW_THRESHOLD = 0.02

# RESOLUTION MERGING
# ------------------------------------------------------------------------
RES_MERGE = IMPROPHE

# CO-REGISTRATION OPTIONS
# ------------------------------------------------------------------------
DIR_COREG_BASE = NULL
COREG_BASE_NODATA = -9999

# MISCELLANEOUS OPTIONS
# ------------------------------------------------------------------------
IMPULSE_NOISE = FALSE
BUFFER_NODATA = FALSE

# TIER LEVEL
# ------------------------------------------------------------------------
TIER = 1

# PARALLEL PROCESSING
# ------------------------------------------------------------------------
# Multiprocessing options (NPROC, DELAY) only apply when using the batch
# utility force-level2. They are not used by the core function force-l2ps.
# ------------------------------------------------------------------------
NPROC = 1
NTHREAD = 2
PARALLEL_READS = FALSE
DELAY = 3
TIMEOUT_ZIP = 30

# OUTPUT OPTIONS
# ------------------------------------------------------------------------
OUTPUT_FORMAT = GTiff
OUTPUT_DST = FALSE
OUTPUT_AOD = FALSE
OUTPUT_WVP = FALSE
OUTPUT_VZN = FALSE
OUTPUT_HOT = FALSE
OUTPUT_OVV = TRUE

++PARAM_LEVEL2_END++

-Specifications:

I am using FORCE v. 3.5.2
Ubuntu 20.04.6 LTS with 8 Threads and 32 GB RAM

thanks for your cooperation

**Comment by davidfrantz** [comment, marked as answer]
Hi @alwadey,

there are probably two things going on here. The "cannot parse parameter file" issue is likely related to this: https://github.com/davidfrantz/force/discussions/84

Please try the proposed solutions in that thread, and also consider using Docker or Singularity instead of actually installing FORCE.

The second thing is that there is no output because your image is completely cloudy. If you really want to process this image, use ``MAX_CLOUD_COVER_FRAME = 100``.

Cheers,
David

**Reply by alwadey** [reply]
HI a@David Frantz and thanks.
Actually, it works when adjust prm file MAX_CLOUD_COVER_FRAME = 100, however, in metadata, cloud cover is zero and not 100%.
